resources/css/common.css

```diff
@@ -111,6 +111,12 @@ footer {
 	clear: both;
 }
 
+.clipboard {
+	position: absolute;
+	right: 0;
+  cursor: pointer;
+}
```

Comment:
Fix the indentation of cursor, and remove position and right properties; that way the button will be more visible.